README.md

```diff
@@ -35,7 +35,8 @@ https://akeegandev.github.io/math_magicians/
 
 ## 🤝 Contributing
 
-Contributions, issues, and feature requests are welcome!
+👤 **Boaz Aduda**
+- GitHub: [@Aduda-Boaz](https://github.com/Aduda-Boaz)
 
```

Comment:
- [ ] **Optional:** Kindly add the testing section to your `readme.me` to make it more professional.